packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -482,6 +493,28 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
         entry.handleP.resolve(this.getBlobHandle(localId));
     }
 
+    private getTimeInfo(pendingEntry: PendingBlob, eventName: string, noTTLEventName: string) {
```

Comment:
maybe rename to logTimeInfo? as the get prefix makes me think this will return something